xml/dm_registers.xml

```diff
@@ -340,13 +340,17 @@ same project unless stated otherwise.
             state change has completed.  Hardware may
             take an arbitrarily long time to complete activation or deactivation and will
             indicate completion by setting {dmcontrol-dmactive} to the requested value.
+            During this time, the DM may ignore any register writes.
 
             <value v="0" name="inactive">
             The module's state, including authentication mechanism,
             takes its reset values (the {dmcontrol-dmactive} bit is the only bit which can
             be written to something other than its reset value). Any accesses
             to the module may fail. Specifically, {dmstatus-version} might not return
             correct data.
+
+            When this value is written, the DM may ignore any other bits written
```

Comment:
```suggestion
            When this value is written, the DM should ignore any other bits written
```
AFAIU, this should be required from the DM. Otherwise it's unclear how this is an exception to: https://github.com/riscv/riscv-debug-spec/blob/ca36be78dd0ddb0dafa974ce554aff42b796d679/debug_module.adoc?plain=1#L371-L373

Reply:
I think this is already consistent. The original text said:
> Don't change hartsel when an abstract command is executing.
This allows a hardware implementation to not have to deal with that confusing case.

The new text says:
> Don't change hartsel when an abstract command is executing, unless dmcontrol is written to 0.
The case for the hardware still isn't confusing because dmactive=0 will cancel the abstract command somehow. Or the hardware may choose to wait until the command completes, in which case it can easily do so by ignoring the change to hartsel.

All this accommodation is to give the hardware implementation options, while allowing the debugger to reset without reading hartsel first.

Reply:
But thinking about this more, we need a statement that says it's OK to change hartsel (and maybe relax other constraints) when writing dmcontrol=0. Maybe saying should like you suggested is the simplest way to do that.

Reply:
I think this is resolved more cleanly by saying hardware should ignore changes to hartsel while an abstract command is being executed.